### DISCUSSION TITLE
how to escape special characters from automatic transformation

### DISCUSSION THREAD

**Discussion by snatch2013** [requarks/wiki#4460] [Help / Questions]
Hello,

sorry for a dumb question. Is it possible somehow to escape special characters, for example, underscore, from automatic transformation in wysiwig editor? I tried to find this information in the user manual, but the chapter about wysiwig editor is in "coming soon" stage :(
 Thanks in advance!

Regards,
Vladimir

**Comment by snatch2013** [comment]
hm, strange, nobody uses special characters in texts? :(

**Comment by NGPixel** [comment]
Do you have a example screenshot of the issue?

**Reply by snatch2013** [reply]
It is impossible to show on screenshot what is absent :) 
For example, I am starting to type
![image](https://user-images.githubusercontent.com/3762271/139851315-0d4c6e5b-160e-47ea-89a6-cbd7f208f7fa.png)

and after I type a second underscore symbol the editor removes both symbols and converts the string between them to italic.
![image](https://user-images.githubusercontent.com/3762271/139851513-20e0b591-9666-4d3e-bb86-20c574339892.png)

**Reply by GesiIvy128** [reply]
I encountered the same issue! In Markdown mode, when I type mathematical formulas, underscores cause incorrect italic formatting in my source code! See the screenshot below
![2024-03-18_005614](https://github.com/requarks/wiki/assets/16591673/024f6594-033a-4394-81c8-7a38476a294c)

**Comment by prplecake** [comment]
Yikes, this was raised nearly three years ago?

Why would the WYSIWYG editor use markdown formatting? If I wanted that I'd use the Markdown editor, which I'll probably end up having to do since the wysiwyg editor doesn't let us escape it.

It's impossible to write something like "E_ACTIVE_LOCKOUT" in the WYSIWYG editor without it turning into "EACTIVELOCKOUT" (with ACTIVE being italicized. Correctly, the github editor doesn't fuck it up.
